scripts/generate.sh

```diff
@@ -0,0 +1,19 @@
+# Get latest OpenAPI specification
+git clone https://gitlab.zalopay.vn/accounting/dojo/docusaurus-playground.git
+
+# Generate models from specification
+openapi-generator-cli generate \
+		-i docusaurus-playground/specs/en/* \
+		-g typescript-node \
+		-t templates/typescript \
+		-o build \
+		--global-property models,supportingFiles \
+		--additional-properties=serviceName=Tokenization \
+		--additional-properties=modelPropertyNaming=original \
+		--additional-properties=helperFunctions=./templates/helpers/camelCase.ts
+
+cp build/model/* src/zalopay/models
```

Comment:
@noradomi should we remove existing files in `src/zalopay/models` before copying?

Reply:
Also, there's a directory named `src/zalopay/model`. Any better name for `src/zalopay/models`